spec/controllers/items_controller.rb

```diff
@@ -62,4 +62,24 @@
     end
   end
 
+  describe '#destroy' do
+    let(:item) { create(:item) }
+    subject { delete :destroy, params: { id: item.id } }
+
+    it 'should redirect to home' do
+      expect(subject).to redirect_to(root_path)
+    end
+
+    it 'should redirect with notice' do
+      subject
+      expect(flash[:notice]).to be_present
+    end
+
+    it 'should destroy author' do
+      item_id = item.id
```

Comment:
What does that line do? I don't see item_id used anywhere.

Reply:
It's item id(paramater) for delete method. Without this line destroy action will not delete item.

Reply:
But you're not passing this variable to destroy method. `id: item`, not `item_id`. That's why I asked.

Reply:
It's for creating item with let. I changed this lane to be more clear.